README.md

```diff
@@ -86,15 +86,15 @@ and the data files in `/tmp/share`.
 
  ## Playing the game
  
- The game takes place on a rectangular areana where a snake continuously
+ The game takes place on a rectangular arena where a snake continuously
  move in one of the four directions: left, right, up and down --- it never 
- stops. As the snake moves it looses energy and if all of it is exausted, the
+ stops. As the snake moves, it looses energy, and if all this energy is exausted, the
```

Comment:
Those comms are probably not necessary (as they would be in portuguese). Furthermore, "looses" should be "loses" and "exausted" should be "exhausted".

```suggestion
 stops. As the snake moves it loses energy and if all this energy is exhausted, the
```

Reply:
I think we should let the second comma stay there. It makes the readability more fluid, since you are saying "energy" again (I'm not a grammar expert btw). What you think?
>stops. As the snake moves it loses energy, and if all this energy is exhausted, the

Reply:
I'm not against it